hw/ip/aes/doc/checklist.md

```diff
@@ -51,6 +51,9 @@ Code Quality  | [LINT_PASS][]           | Done        |
 Code Quality  | [CDC_SETUP][]           | N/A         |
 Code Quality  | [FPGA_TIMING][]         | Done        |
 Code Quality  | [CDC_SYNCMACRO][]       | N/A         |
+Security      | [SEC_CM_IMPLEMENTED][]  | Not Started |
```

Comment:
@vogelpi this highlights that we shouldn't be `D2` for AES, but I think there are other reasons why AES should still be `D1`. WDYT? If you back out to `D1` you should mark all these as `Not started`

Reply:
This depends a bit on which version we are talking about. Back in Nov, we were targeting 0.5 (plain vanilla AES) and all the points in `D2` were fulfilled already. Thus we went for `D2`. But I agree, now that we are talking about security hardening, it is definitely not `D2`. The right thing to do is probably to also bump the version when going back to `D1`. I will do that once this PR is in.

Reply:
sounds good. could switch to Version 1.0 and revert to D1 and mark all these as not started in a separated PR.